src/python/pants/backend/python/goals/export.py

```diff
@@ -50,6 +50,17 @@ def debug_hint(self) -> str | None:
         return self.resolve
 
 
+@union
+class ExportPythonToolSentinel:
+    """Pythion tools use this as an entry point to say how to export their tool virtualenv."""
```

Comment:
```suggestion
    """Python tools use this as an entry point to say how to export their tool virtualenv."""
```

Reply:
It would probably be good to add instructions in a new paragraph. Subclass this class then create a rule going from the sub class to ExportPythonTool. Register the union rule. If the tool is in pantsbuild/pants, update the export test.